spec/controllers/hyku/registrations_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe Hyku::RegistrationsController, type: :controller do
+  before do
+    allow(Settings.devise).to receive(:account_signup).and_return(account_signup_enabled)
+    # Recommended by Devise: https://github.com/plataformatec/devise/wiki/How-To:-Test-controllers-with-Rails-3-and-4-%28and-RSpec%29
+    @request.env['devise.mapping'] = Devise.mappings[:user]
```

Comment:
I'm fairly certain we don't need that.  That's for when you have more than one mapping.